components/sensor/ld2410.rst

```diff
@@ -25,58 +25,279 @@ Use of hardware UART pins is highly recommended, in order to support the out-of-
+        detection_distance:
+          name: Detection Distance
+        g0:
+          move_energy:
+            name: g0 move energy
+          still_energy:
+            name: g0 still energy
+        g1:
+          move_energy:
+            name: g1 move energy
+          still_energy:
+            name: g1 still energy
+        g2:
+          move_energy:
+            name: g2 move energy
+          still_energy:
+            name: g2 still energy
+        g3:
+          move_energy:
+            name: g3 move energy
+          still_energy:
+            name: g3 still energy
+        g4:
+          move_energy:
+            name: g4 move energy
+          still_energy:
+            name: g4 still energy
+        g5:
+          move_energy:
+            name: g5 move energy
+          still_energy:
+            name: g5 still energy
+        g6:
+          move_energy:
+            name: g6 move energy
+          still_energy:
+            name: g6 still energy
+        g7:
+          move_energy:
+            name: g7 move energy
+          still_energy:
+            name: g7 still energy
+        g8:
+          move_energy:
+            name: g8 move energy
+          still_energy:
+            name: g8 still energy
+
+.. _ld2410-sensors:
+
+Configuration variables:
+************************
+
+- **light** (*Optional*, int): When in :ref:`engineering mode<ld2410-engineering-mode>`, indicates the light sensitivity, otherwise ``unknown``.
+  Value between ``0`` and ``255`` inclusive. Thought it seems that the value ``85`` is the lowest value at complete darkness.
+  Notice that this sensor is not very reliable. A more suitable alternative will be :ref:`BH1750<BH1750>`.
+  All options from :ref:`Sensor <config-sensor>`.
+- **moving_distance** (*Optional*, int): Distance in cm of detected moving target.
+  All options from :ref:`Sensor <config-sensor>`.
+- **still_distance** (*Optional*, int): Distance in cm of detected still target.
+  All options from :ref:`Sensor <config-sensor>`.
+- **moving_energy** (*Optional*, int): Energy for moving target.
+  Value between ``0`` and ``100`` inclusive.
+  All options from :ref:`Sensor <config-sensor>`.
+- **still_energy** (*Optional*, int): Energy for still target.
+  Value between ``0`` and ``100`` inclusive.
+  All options from :ref:`Sensor <config-sensor>`.
+- **detection_distance** (*Optional*, int): Distance in cm of target.
+  All options from :ref:`Sensor <config-sensor>`.
+- **gX_move_energy** (*Optional*, int): When in :ref:`engineering mode<ld2410-engineering-mode>`, the move energy of the Xth gate(X => 0 to 8), otherwise ``unknown``.
+  Value between ``0`` and ``100`` inclusive.
+  All options from :ref:`Sensor <config-sensor>`.
+- **gX_still_energy** (*Optional*, int): When in :ref:`engineering mode<ld2410-engineering-mode>`, the still energy of the Xth gate(X => 0 to 8), otherwise ``unknown``.
+  Value between ``0`` and ``100`` inclusive.
+  All options from :ref:`Sensor <config-sensor>`.
```

Comment:
The structure here probably needs to change to match the layout changes

Reply:
I thought about it, but not sure how to structure it, can you please suggest an example?

Reply:
done

Reply:
Maybe something like this?
```suggestion
- **gX** (*Optional*): When in :ref:`engineering mode<ld2410-engineering-mode>`, the Xth gate(X => 0 to 8) energy information.
  
  - **move_energy** (*Optional*): The move energy of the gate.
    Value between ``0`` and ``100`` inclusive.
    All options from :ref:`Sensor <config-sensor>`.
  - **still_energy** (*Optional*): The still energy of the gate.
    Value between ``0`` and ``100`` inclusive.
    All options from :ref:`Sensor <config-sensor>`.
```

Reply:
Haha just did something similiar, it is already available in preview